README.md

```diff
@@ -1,34 +1,19 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+A graphical device tree visualizer (early development) 
 
-## Getting Started
+## Local Deployment
+
+Install wasm-pack
```

Comment:
let's put the `cargo install` command here, and add npm as well

something like

### Dependencies

Install `wasm-pack` and npm dependencies:

```sh
cargo install wasm-pack
npm install
```

### Run the app in development mode

```sh
npm start
```